examples/js-console/src/main/webapp/index.html

```diff
@@ -20,6 +20,9 @@
     <button onclick="output(keycloak.createLoginUrl())">Show Login URL</button>
     <button onclick="output(keycloak.createLogoutUrl())">Show Logout URL</button>
     <button onclick="output(keycloak.createRegisterUrl())">Show Register URL</button>
+    <p>
```

Comment:
The reason for this ```textarea``` is that you can paste any token string to check how the endpoint behaves. Useful when you want to introspect and check previously issued tokens.